### ISSUE TITLE
[Feature Request] Add "Already watched" option.

### ISSUE THREAD

**Issue by v2maxx** [yuliskov/SmartTubeNext#1843]
Adding the youtube-app option "already watched", will give the opportunity to get rid of videos that one has already watched and declutter the feed from duplicate suggestions, without having to alter the algorithm by choosing "not interested".

**Comment by Amandaville** [comment]
Agreed, my home screen/recommended section is extremely cluttered with watched videos.  Fix that and SmartTubeNext will be the perfect YT experience.
